### PR TITLE
RemoveLoopFromALinkedList.cpp

### PR DESCRIPTION
**Pull request by AnushkaS09**
ISSUE: #1339
Solved in C++(as assigned)

### CODE REVIEW

**Comment by rudrakshi99** [comment]
Follow proper guidelines link 4 space indentations, test cases, brief intro, etc.

**Comment by AnushkaS09** [comment]
[@plazzy99] @tarun26091999 @rudrakshi99  Kindly review

File: Linked_list/readme.md [hunk 1]
```diff
@@ -34,7 +34,6 @@ properties.
 * Splitting ----> [C++](/Code/C++/Splitting_Linked_lists.cpp)
 * Implementation of Linked List ----> [Python](/Code/Python/linked_list.py)
```
Comment (rish-singhal) [inline Linked_list/readme.md:35]: sort this list in lexicographic order

**Review by rish-singhal — CHANGES_REQUESTED** [review]

**Comment by tarun26091999** [comment]
@rish-singhal look into it.

**Comment by rish-singhal** [comment]
@tarun26091999 looks good to me.

**Pull request merged**